samples/bookinfo/src/build-services.sh

```diff
@@ -26,48 +26,64 @@ VERSION=$1
 PREFIX=$2
 SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
+# Docker build variables
+ENABLE_MULTIARCH_IMAGES=${ENABLE_MULTIARCH_IMAGES:-"false"}
+
+if [ "${ENABLE_MULTIARCH_IMAGES}" == "true" ]; then
+  PLATFORMS="linux/arm64,linux/amd64"	
+  DOCKER_BUILD_ARGS="docker buildx build --platform ${PLATFORMS} --push"
+  # Install QEMU emulators
+  docker run --rm --privileged tonistiigi/binfmt --install all
+  docker buildx rm multi-builder || :
+  docker buildx create --use --name multi-builder --platform ${PLATFORMS}
+  docker buildx use multi-builder
+else
+  DOCKER_BUILD_ARGS="docker build"	
+fi
+
 pushd "$SCRIPTDIR/productpage"
-  docker build --pull -t "${PREFIX}/examples-bookinfo-productpage-v1:${VERSION}" -t "${PREFIX}/examples-bookinfo-productpage-v1:latest" .
+  ${DOCKER_BUILD_ARGS} --pull -t "${PREFIX}/examples-bookinfo-productpage-v1:${VERSION}" -t "${PREFIX}/examples-bookinfo-productpage-v1:latest" .
   #flooding
-  docker build --pull -t "${PREFIX}/examples-bookinfo-productpage-v-flooding:${VERSION}" -t "${PREFIX}/examples-bookinfo-productpage-v-flooding:latest" --build-arg flood_factor=100 .
+  ${DOCKER_BUILD_ARGS} --pull -t "${PREFIX}/examples-bookinfo-productpage-v-flooding:${VERSION}" -t "${PREFIX}/examples-bookinfo-productpage-v-flooding:latest" --build-arg flood_factor=100 .
 popd
 
 pushd "$SCRIPTDIR/details"
   #plain build -- no calling external book service to fetch topics
-  docker build --pull -t "${PREFIX}/examples-bookinfo-details-v1:${VERSION}" -t "${PREFIX}/examples-bookinfo-details-v1:latest" --build-arg service_version=v1 .
+  ${DOCKER_BUILD_ARGS} --pull -t "${PREFIX}/examples-bookinfo-details-v1:${VERSION}" -t "${PREFIX}/examples-bookinfo-details-v1:latest" --build-arg service_version=v1 .
   #with calling external book service to fetch topic for the book
-  docker build --pull -t "${PREFIX}/examples-bookinfo-details-v2:${VERSION}" -t "${PREFIX}/examples-bookinfo-details-v2:latest" --build-arg service_version=v2 \
+  ${DOCKER_BUILD_ARGS} --pull -t "${PREFIX}/examples-bookinfo-details-v2:${VERSION}" -t "${PREFIX}/examples-bookinfo-details-v2:latest" --build-arg service_version=v2 \
 	 --build-arg enable_external_book_service=true .
 popd
 
+
 pushd "$SCRIPTDIR/reviews"
   #java build the app.
   docker run --rm -u root -v "$(pwd)":/home/gradle/project -w /home/gradle/project gradle:4.8.1 gradle clean build
```

Comment:
for review, we need to consider following
```
Dockerfile:15
--------------------
  13 |     #   limitations under the License.
  14 |
  15 | >>> FROM websphere-liberty:20.0.0.6-full-java8-ibmjava
  16 |
  17 |     ENV SERVERDIRNAME reviews
--------------------
error: failed to solve: rpc error: code = Unknown desc = websphere-liberty:20.0.0.6-full-java8-ibmjava: no match for platform in manifest sha256:57fc2ac9b00c23f89fe3e3c9208cc6183bbd93d7425849436df7f2aa56144021: not found
```

Reply:
worked around it https://github.com/istio/istio/pull/37546/commits/bea2c17e51c9b16824d2bbb7b15c017fcc297e9e